controllers/homePageAPI.js

```diff
@@ -1,17 +1,20 @@
 const express = require('express');
-const homePage = require('../models/homePage')
-const view = require('../views/homePageView');
 const router = express.Router();
 
 router.get('/', (req, res) =>
 {
 	console.log("Receieved req for homePage listings");
-	homePage.GetRecentListings(
-		(err, results) =>
-		{ 
-			if (err) throw err;
-			res.sendFile(view.viewHomePage(results));
-		});
+	res.render('pages/home')
+});
+
+router.get('/confirmPurchase', (req, res) =>
```

Comment:
why did we add a confirm purchase and payment summary here in the homePage api?, these should be on 'listingAPI' controller class